src/index.php-maintenance-message

```diff
@@ -1,4 +0,0 @@
-<h2>Site Maintenance In Progress</h2>
```

Comment:
In theory this file still has value to me.  I have done a number of db migrations in the past where I simply:
```
cp index.php index.php-real
cp index.php-maintenance-message index.php
[do the database migration while the site is in maintenance mode]
mv index.php-real index.php
```

If you have a different ideal on how to do this, please let me know.  This file is so small and simple that we can easily delete it now and recreate it if it's needed again.